recipes/buildpy/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "buildpy" %}
+{% set version = "6.2.4" %}
+{% set sha256 = "018443ed7dccf68572f97b31b06645b6d535281747f72edca38af9ed44c8f01f" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://files.pythonhosted.org/packages/a9/4d/c4a1a46a34fcd285d15d93d32f8daaf5c0f28a72adde156897dbd527a3b0/buildpy-6.2.4.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vvv"
+  skip: True
```

Comment:
This doesn't actually build anything.  You should remove this line.

Reply:
Thank you for your comment. I removed `skip: True`.

Reply:
I replaced the line with `skip: True  # [win or py2k]`.